code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3131,47 +3131,46 @@ datum
 					M.Jitter(5)
 					if(adj_temp > 0 && holder.has_reagent("frostoil"))
 						holder.remove_reagent("frostoil", 10*REAGENTS_METABOLISM)
-
 					holder.remove_reagent(src.id, 0.1)
-				icecoffee
-					name = "Iced Coffee"
-					id = "icecoffee"
-					description = "Coffee and ice, refreshing and cool."
-					color = "#102838" // rgb: 16, 40, 56
-					adj_temp = -5
-
-				soy_latte
-					name = "Soy Latte"
-					id = "soy_latte"
-					description = "A nice and tasty beverage while you are reading your hippie books."
-					color = "#664300" // rgb: 102, 67, 0
-					adj_sleepy = 0
-					adj_temp = 5
+			icecoffee
+				name = "Iced Coffee"
+				id = "icecoffee"
+				description = "Coffee and ice, refreshing and cool."
+				color = "#102838" // rgb: 16, 40, 56
+				adj_temp = -5
 

```

Comment:
All these indentation changes effectively eleminates these reagents from inheriting any traits of regular coffee, I believe. Please correct me if I'm wrong.